inline-lexer.js

```diff
@@ -112,6 +112,29 @@ InlineLexer.prototype.next = function next(type, text, scanner) {
             }
             this.accumulator += c;
             this.type = 'alphanum';
+        } else if (scanner.leader === '!' && (c === "'" || c === '"')) {
+            this.type = 'string';
+            i++;
+            while (i < text.length) {
+              d = text[i];
+              if (d === '\\' && ++i < text.length) {
```

Comment:
I think it would be sufficient for now allow escapes for single quote only. We can treat all other escapes as a syntax error, like `\n`, which would break the runtime engine.